### PR TITLE
Resolutions

### PR DESCRIPTION
**Pull request by thecalliopecrashed**
This removes the coordinator registration page.
Instead of registering directly, we should now use:

```bash
docker-compose exec web rails c
> Coordinator.create!(
>   email: "grace.c.youngblood@gmail.com",
>   name: "Grace C Youngblood",
>   password_digest: BCrypt::Password.create("password"),
>   uuid: SecureRandom.uuid,
> )
```

Equally important, this introduces the `Resolution` database model.
We'll start to tie these together for the coordinator.

### CODE REVIEW

**Comment by thecalliopecrashed** [comment]
This most recent commit keeps everybody logged in until they click "Log out", even when they refresh the page.

It also makes sure every coordinator can view every participant's information.

Recommended that we deploy these changes right away.

**Comment by thecalliopecrashed** [comment]
remember_login

File: src/Account.js [hunk 1]
```diff
@@ -50,7 +50,14 @@ class Account {
         ) == ${JSON.stringify(password)} ?
         ${this.model}.find_by(JSON.parse('${JSON.stringify(attributes)}')).uuid :
         {}
-    `.then(response => response.json().then(uuid => this.watch(uuid) ))
+      `.then(response =>
+        response
+        .json()
+        .then(uuid => {
+          localStorage.setItem(session_uuid, uuid)
```
Comment (campbellsean) [inline src/Account.js]: Ahh nice

File: src/components/CoordinatorLogin.js [hunk 2]
```diff
@@ -39,10 +38,6 @@ const CoordinatorLogin = observer(({assembly}) => (
       </Button>
     </Form>
 
-    <InternalLink to={CoordinatorRegister} assembly={assembly} >
```
Comment (campbellsean) [inline src/components/CoordinatorLogin.js]: Great!

File: src/components/CoordinatorMenu.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
+import React from "react"
```
Comment (campbellsean) [inline src/components/CoordinatorMenu.js]: Fantastic

**Pull request closed without merging**